### ISSUE TITLE
Accessibility - Movie Poster Alt Text

### ISSUE THREAD

**Issue by JarenGarman** [JarenGarman/rancid-tomatillos#39]
## Problem

- WAVE test reveals that our movie posters all share the same alt text
  - Because we don't list the movie title in the poster, this means that a user with a screen reader or poor internet that cannot download images will not be able to know which movie the poster is associated with

## Remedy

- Change the JS to give each poster a unique alt text property that is tied to the movie title

![Movie Poster Alt Text Issue](https://github.com/user-attachments/assets/f67a400f-a0a7-4a39-8aef-0074e0cae1a7)
